s/elements/stickpad/setups/setup-stickpad-events.ts

```diff
@@ -0,0 +1,22 @@
+import {attachEvents} from "../../../tools/attach-events.js"
+import {StickpadStarters} from "../types.js"
+
+export function setupStickpadEvents(
+	{
+	stickPad,
+	setStick,
+	setCenterPosition
+	}: StickpadStarters) {
+
+	return () => {
+		attachEvents(window, {
+			pointerup() {setStick(false)}
+		})
+		attachEvents(stickPad, {
+			pointerdown(e) {
+				const ev = <PointerEvent>e
+				setCenterPosition(ev)
+			}
+		})
+	}
```

Comment:
`attachEvents` returns a disposer function.

`use.setup` is expecting you to return a disposer function.

this function needs to return a function that calls both disposers from the two `attachEvents` calls.

the disposer function is what allows the event listeners to be cleaned up when the element is disconnected from the dom.